I can't to reproduce the pylint mqt error in t2d yet... but the output was:

``` bash
======== Testing test_flake8 ========
runbot_travis2docker/models/runbot_build.py:57:80: E501 line too long (80 > 79 characters)
runbot_travis2docker/models/runbot_build.py:59:80: E501 line too long (102 > 79 characters)
runbot_travis2docker/models/runbot_build.py:61:80: E501 line too long (81 > 79 characters)
runbot_travis2docker/models/runbot_build.py:64:17: E128 continuation line under-indented for visual indent
runbot_travis2docker/models/runbot_build.py:70:80: E501 line too long (80 > 79 characters)
runbot_travis2docker/models/runbot_build.py:72:80: E501 line too long (84 > 79 characters)
runbot_travis2docker/models/runbot_build.py:76:39: E201 whitespace after '{'
runbot_travis2docker/models/runbot_build.py:76:80: E501 line too long (88 > 79 characters)
runbot_travis2docker/models/runbot_build.py:80:80: E501 line too long (156 > 79 characters)
runbot_travis2docker/models/runbot_build.py:85:80: E501 line too long (144 > 79 characters)
runbot_travis2docker/models/runbot_build.py:101:34: E127 continuation line over-indented for visual indent
runbot_travis2docker/models/runbot_build.py:102:80: E501 line too long (93 > 79 characters)
runbot_travis2docker/models/runbot_repo.py:12:80: E501 line too long (83 > 79 characters)
======== Testing test_pylint ========
extra_params_cmd ['--extra-params', '--load-plugins=pylint_odoo']
************* Module runbot_travis2docker.models.runbot_build
runbot_travis2docker/models/runbot_build.py:62: [W0703(broad-except), RunbotBuild._get_introspection] Catching too general exception Exception
runbot_travis2docker/models/runbot_build.py:75: [W0703(broad-except), RunbotBuild._get_introspection_html] Catching too general exception Exception
```

--- a/runbot_travis2docker/models/runbot_build.py
+++ b/runbot_travis2docker/models/runbot_build.py
@@ -54,37 +54,12 @@ class RunbotBuild(models.Model):
     @api.depends('state')
     def _get_introspection(self):
         for build in self:
-            url = "http://%s/instance_introspection.json" % (build.domain + '.')
+            url = "http://%s/instance_introspection.json" % (build.domain)
             try:
-                build.introspection = str(json.dumps({'status': 'fail', 'error': 'Not computed yet'}))
-                if build.dest and build.domain.find('False') < 0:
-                    build.introspection = str(json.loads(requests.get(url).text))
-            except Exception as e:
-                build.introspection = str(json.dumps({'status': 'failV',
-                'error': e.message, 'url': url}))
-
-    @api.depends('introspection')
-    def _get_introspection_html(self):
-        for build in self:
-
-            url = "http://%s/instance_introspection.json" % (build.domain + '.')
-            try:
-                introspection_dict = {'status': 'fail', 'error': 'Not computed yet'}
-                if build.dest and build.domain.find('False') < 0:
-                    introspection_dict = json.loads(requests.get(url))
-            except Exception as e:
-                introspection_dict = { 'status': 'fail', 'error': e.message, 'url': url}
-            introspection_html = ''
-            if isinstance(introspection_dict, dict):
-                for key in introspection_dict:
-                    introspection_html += '<br/><span class="json-key">%s</span>: <span class="json-value">%s</span>,<br/>' % (key, introspection_dict[key])
-            if isinstance(introspection_dict, list):
-                for el in introspection_dict:
-                    introspection_html += "}<br/>{"
-                    for key in el:
-                        introspection_html += '<br/><span class="json-key">%s</span>: <span class="json-value">%s</span>,<br/>' % (key, el[key])
-
-            build.introspection_html = introspection_html
+                value = requests.get(url)
+                build.introspection = value.text
+            except:
+               pass
 
     dockerfile_path = fields.Char(
         help='Dockerfile path created by travis2docker')
@@ -99,7 +74,6 @@ class RunbotBuild(models.Model):
     branch_short_name = fields.Char(help='Branch short name e.g. pull/1, 8.0')
     introspection = fields.Text(help='Introspection', store=True,
                                  compute='_get_introspection')
-    introspection_html = fields.Html(help='Introspection', compute='_get_introspection_html')
 
     def get_docker_image(self, branch_closest=None):
         self.ensure_one()

--- a/runbot_travis2docker/models/runbot_build.py
+++ b/runbot_travis2docker/models/runbot_build.py
@@ -59,7 +59,7 @@ class RunbotBuild(models.Model):
                 value = requests.get(url)
                 build.introspection = value.text
             except:
-               pass
+                build.introspection = ''
 
     dockerfile_path = fields.Char(
         help='Dockerfile path created by travis2docker')

--- a/runbot_travis2docker/models/runbot_build.py
+++ b/runbot_travis2docker/models/runbot_build.py
@@ -5,7 +5,6 @@
 
 import logging
 import os
-import json
 import requests
 import subprocess
 import sys
@@ -73,7 +72,7 @@ class RunbotBuild(models.Model):
     is_pull_request = fields.Boolean(help="True is a pull request.")
     branch_short_name = fields.Char(help='Branch short name e.g. pull/1, 8.0')
     introspection = fields.Text(help='Introspection', store=True,
-                                 compute='_get_introspection')
+                                compute='_get_introspection')
 
     def get_docker_image(self, branch_closest=None):
         self.ensure_one()

